### ISSUE TITLE
Wrong Balance after staking EOS

### ISSUE THREAD

**Issue by filipealonso** [greymass/eos-voter#160]
Updating Staked EOS doesn't update Liquid in "Token Balances" tab and therefore the Total sum is also wrong.

Example:

1) Before updating staked EOS:
Liquid: 10
Staked: 10
Total: 20

2) Stake the remaining 10 liquid EOS 

3) After updating staked EOS:
Liquid: 10 <-- WRONG
Staked: 20
Total: 30 <-- WRONG

Probably you just need to refresh the view, but check it.

**Comment by aaroncox** [comment]
A balance update after staking is triggered, but maybe there's a bug preventing this. 

I've heard this report a couple times now and have been unable to reproduce, but we will continue looking.

**Comment by dafuga** [comment]
Hey @filipealonso, thanks for reporting this! I have been unable to reproduce the issue though. Do you know what version of the app you were using? Are you experiencing the same issue with the latest build? Thanks!

**Comment by filipealonso** [comment]
@dafuga it was with version 0.2.1 if I'm not mistaken. I had been updating versions consecutively as you were releasing them. Before that I remember that the wallet state (introduced with 0.2.0 right?) was undefined (meaning i was already "logged in" since 0.14 or 0.15 but hadn't run the full setup process yet) but then I completed the setup process in that version and proceeded to stake some EOS, which is when I saw this bug.

Since then I haven't re-staked EOS with your wallet so I cannot confirm if the problem persists.

Thank you for looking into it ;)
